core/src/service/usServiceHooks.cpp

```diff
@@ -52,15 +54,15 @@ std::shared_ptr<ServiceListenerHook> ServiceHooks::AddingService(const ServiceRe
   {
     lh->Added(coreCtx->listeners.GetListenerInfoCollection());
   }
-  catch (const std::exception& e)
+  catch (const std::exception& )
```

Comment:
Because we capture the exception pointer now, we can get rid of the `catch (const std::exception&)` block and only use the catch all block.